selfdrive/car/gm/carstate.py

```diff
@@ -33,7 +33,8 @@ def update(self, pt_cp, cam_cp, loopback_cp):
     self.cruise_buttons = pt_cp.vl["ASCMSteeringButton"]["ACCButtons"]
     self.buttons_counter = pt_cp.vl["ASCMSteeringButton"]["RollingCounter"]
     self.pscm_status = copy.copy(pt_cp.vl["PSCMStatus"])
-    self.moving_backward = pt_cp.vl["EBCMWheelSpdRear"]["MovingBackward"] != 0
+    # 0 - unknown, 1 - forward, 2 - reverse, 3 - unsupported, 4 - fault
```

Comment:
```suggestion
    # This is to avoid a fault where you engage while still moving backwards after shifting to D.
    # An Equinox has been seen with an unsupported status, so only check if either wheel is in reverse (2)
```